internal/ingress/controller/controller.go

```diff
@@ -130,6 +130,9 @@ func (n *NGINXController) syncIngress(interface{}) error {
 	for _, server := range servers {
 		if !hosts.Has(server.Hostname) {
 			hosts.Insert(server.Hostname)
+			if server.Alias != "" {
+				hosts.Insert(server.Alias)
+			}
 		}
```

Comment:
This can result in duplicate `server.Alias`'s in `hosts` collection. I think this `if` should be out of `if !hosts.Has(server.Hostname) {` and similar to that, like:

```
		if !hosts.Has(server.Hostname) {
			hosts.Insert(server.Hostname)
		}
		if server.Alias != "" && !hosts.Has(server.Alias) {
			hosts.Insert(server.Alias)
		}
```

Reply:
Updated.

Reply:
Hi, seems you missed this part.
The code still has `if server.Alias != "" && !hosts.Has(server.Hostname) {` instead of `if server.Alias != "" && !hosts.Has(server.Alias) {`

Currently, the alias would not be added to the `hosts` list because `server.Hostname` would be added in the previous line and `host.Has(server.Hostname)` would always be false.

Reply:
@startnow65 thank you for noticing that error.

Reply:
Good catch @startnow65!

Reply:
Oops. Sorry about that.

I have never had to add a commit to a branch that was already merged.
I added the fix but I do not see the update here. Do I need to create a new PR?

Reply:
@walkafwalka no, I already fixed this in another PR.